Kitodo/src/main/webapp/WEB-INF/templates/includes/processFromTemplate/dataEdit.xhtml

```diff
@@ -18,6 +18,9 @@
         xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
         xmlns:p="http://primefaces.org/ui">
 
+    <div style="font-weight:bold;padding:.75em 1.5em;">
```

Comment:
Please move this styling to a separate CSS class and use `padding: var(--default-half-size) var(--default-full-size);` instead of `em`.